### ISSUE TITLE
Comment in the toFixed() method

### ISSUE THREAD

**Issue by unmeshvrije** [yousifAlneamy/my-web-page#1]
`Function.prototype.method = function (name, func) { // here we're augmenting Function.prototype so we can add/make a method available to all functions
        this.prototype[name] = func; // here we're assigning a function available to all the objects of "this", because we're augmenting "func()" to its prototype property "name"
        return this;
    };`

Usually there are no multiple objects of this. You probably meant all instances of the function. this itself represents an object. Please elaborate/rectify the comment.

**Comment by yousifAlneamy** [comment]
Yes exactly, What I meant is that whatever type of object (Number, String, Boolean, etc) can use this method() to add a function to its prototype property.

"this" will represent (an instance of) the object who called the method, so by that "this keyword" give us the flexibility to use "method()" with any object to add "name()" to its prototype.

"Function.prototype.method" can be misleading, why we write "Function", so my comment was supposed to remove this confusion.

**Comment by yousifAlneamy** [comment]
(all the objects of "this"), in my code, I used Number.prototype, after that, all objects of Number type have the accessibility to my overwritten "toFixed()".

Hope this clear the ambiguity of my "//comment".

**Comment by unmeshvrije** [comment]
You could say "all objects represented by this"!

   Regards,
   Unmesh Joshi

**Comment by yousifAlneamy** [comment]
Right, I will do it. Thanks Unmesh
